src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -22,7 +22,9 @@
     <link rel="import" href="/sys/starcounter.html">
     <link rel="import" href="/sys/starcounter-include/starcounter-include.html">
 
-    <link rel="import" href="/sys/bootstrap.html">
```

Comment:
Since `bootstrap.html` was removed, please also remove the file, the Bootstrap theme and the entry in `bower.json`.